turbinia/api/models/workers_status.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+# Copyright 2022 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Turbinia API server Request models."""
+
+import datetime
+import logging
+
+from datetime import datetime
+from datetime import timedelta
+from operator import itemgetter
+from pydantic import BaseModel
+from typing import ClassVar
+
+from turbinia import state_manager
+from turbinia import config as turbinia_config
+
+log = logging.getLogger('turbinia:api_server:routes:request')
```

Comment:
Should the logger name be changed here I assume?

Reply:
Thanks for noticing that, I fixed it here